api/client.go

```diff
@@ -100,6 +100,13 @@ type ServiceStatus struct {
 // UnitStatus holds status info about a unit.
 type UnitStatus struct {
 	Agent AgentStatus
```

Comment:
That was there, apparently from an early attempt to upgrade from the
original 3 values that seem to be there since 1.16 i think it was
introduced in 1.19 it is not really used except to populate legacy values
bjt since users of the api should expect I decided to leave it untouched.
On Mar 7, 2015 10:04 AM, "Ian Booth" <notifications@github.com> wrote:

> In api/client.go
> <https://github.com/juju/juju/pull/1772#discussion_r25995870>:
>
> > @@ -100,6 +100,13 @@ type ServiceStatus struct {
> >  // UnitStatus holds status info about a unit.
> >  type UnitStatus struct {
> >  	Agent AgentStatus
>
> Is the "Agent" field the legacy agent-state value? Can we add a comment to
> explain what it's used for.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/juju/juju/pull/1772/files#r25995870>.
>